### PR TITLE
fix: rip out microbundle

### PR DESCRIPTION
**Pull request by ParamagicDev**
# Type of PR (feature, enhancement, bug fix, etc.)

:bug: Fix

## Description

Rip out microbundle to provide an unminified, untranspiled stimulus_reflex

## Why should this be added

Make developing easier with sourcemaps and stuff

## Checklist

- [x] My code follows the style guidelines of this project
- [x] Checks (StandardRB & Prettier-Standard) are passing
- [x] This is not a documentation update

Please note that the best way to suggest changes or updates to the [documentation](https://docs.stimulusreflex.com) is to [join Discord](https://discord.gg/XveN625) and leave a note in the #docs channel. Any documentation updates posted as PRs cannot be accepted at this time. :heart:

### CODE REVIEW

File: javascript/package.json [hunk 1]
```diff
@@ -28,18 +28,12 @@
...
   "author": "Nathan Hopkins <natehop@gmail.com>",
-  "source": "./stimulus_reflex.js",
-  "main": "./dist/stimulus_reflex.js",
-  "module": "./dist/stimulus_reflex.module.js",
-  "esmodule": "./dist/stimulus_reflex.modern.js",
+  "main": "./stimulus_reflex.js",
+  "module": "./stimulus_reflex.js",
   "scripts": {
-    "prepare": "yarn build",
-    "postinstall": "node ./scripts/post_install.js",
```
Comment (marcoroth) [inline javascript/package.json:37]: Why did we remove the `postinstall` script here?

File: package.json [hunk 2]
```diff
@@ -28,18 +28,12 @@
...
   "author": "Nathan Hopkins <natehop@gmail.com>",
-  "source": "./javascript/stimulus_reflex.js",
-  "main": "./javascript/dist/stimulus_reflex.js",
-  "module": "./javascript/dist/stimulus_reflex.module.js",
-  "esmodule": "./javascript/dist/stimulus_reflex.modern.js",
+  "main": "./javascript/stimulus_reflex.js",
+  "module": "./javascript/stimulus_reflex.js",
   "scripts": {
-    "prepare": "yarn build",
-    "postinstall": "node javascript/scripts/post_install.js",
```
Comment (marcoroth) [inline package.json:37]: Same here 🙃

File: javascript/package.json (hunk 1 above)
Comment (leastbad) [reply javascript/package.json:37]: good catch. fixed on master.

File: package.json (hunk 2 above)
Comment (leastbad) [reply package.json:37]: fixed on master.

Comment (ParamagicDev) [reply package.json:37]: I got too delete happy apparently!

**Pull request merged**